content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -0,0 +1,453 @@
+        Statsd.countWithTimestamp("example_metric.count_with_timestamp", new Random().nextInt(20), 1205794800, new String[]{"environment:dev"});
+    }
+}
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang="go" >}}
+```go
+package main
+
+import (
+	"log"
+  "time"
+
+	"github.com/DataDog/datadog-go/statsd"
+)
+
+func main() {
+	statsd, err := statsd.New("127.0.0.1:8125")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+  ts := time.Date(2008, time.March, 17, 23, 0, 0, 0, time.UTC)
+	statsd.GaugeWithTimestamp("example_metric.gauge_with_timestamp", 12, []string{"environment:dev"}, 1, ts)
+  statsd.CountWithTimestamp("example_metric.count_with_timestamp", 12, []string{"environment:dev"}, 1, ts)
+}
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang=".NET" >}}
+```csharp
+using StatsdClient;
+
+public class DogStatsdClient
+{
+    public static void Main()
+    {
+        var dogstatsdConfig = new StatsdConfig
+        {
+            StatsdServerName = "127.0.0.1",
+            StatsdPort = 8125,
+        };
+
+        using (var dogStatsdService = new DogStatsdService())
+        {
+            dogStatsdService.Configure(dogstatsdConfig);
+            var random = new Random(0);
+            var dto = new DateTimeOffset(2008, 03, 17, 23, 00, 00, new TimeSpan(0, 0, 0))
+            dogStatsdService.Gauge("example_metric.gauge_with_timestamp", 10, tags: new[] {"environment:dev"}, dto);
+            dogStatsdService.Counter("example_metric.count_with_timestamp", 10, tags: new[] {"environment:dev"}, dto);
+        }
+    }
+}
+```
+{{< /programming-lang >}}
+
+{{< /programming-lang-wrapper >}}
+
+## Ingestion latency
+
+Ingesting historical metrics includes some ingestion latencies. These latencies can be caused by the delay associated with the metric timestamp.
+
+| Metric Delayed by:   | Ingestion Latency                     |
+|----------------------|---------------------------------------|
+| 1-12 hours           | Near Real-Time Ingestion (1 hour MAX) |
+| 12 hours - 30 days   | Up to 14 hour latency                 |
+| +30 days             | +14 hours latency                     |
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_latency.png" alt="Diagram showing how Historical Metrics can take longer to ingest depending on the metric timestamp">}}
+
+## Ingestion billing
+
+Historical Metrics are billed under the Custom Metrics billing SKU. Billable custom metrics are determined by the timestamp of the metrics submitted, regardless of whether they are submitted today or 12 months later. For more information, see the [Custom Metrics billing][3] documentation.
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_billing.png" alt="Diagram showing how Historical Metrics is billed based on the timestamp of the metric" >}}
```

Comment:
This image has too much going on for even me to patiently comb through and understand 😓  and it's a bit misleading because the bolded number of billable custom metrics is only valid for that one hour...but the customer's bill is averaged over all 730 hours in a month